nncf/onnx/graph/metatypes/onnx_metatypes.py

```diff
@@ -709,3 +765,23 @@ def _is_depthwise_conv(model: onnx.ModelProto, node: onnx.NodeProto) -> bool:
     ):
         return True
     return False
+
+
+def _is_embedding(model: onnx.ModelProto, node: onnx.NodeProto) -> bool:
+    """
+    Returns True if the layer can be represented as embedding, False - otherwise.
+
+    :param model: ONNX model to get the node's weight.
+    :param node: Layer to check whether it is embedding.
+    :return: True if the layer is embedding, False - otherwise.
+    """
+    tensor_port_id = ONNXEmbeddingMetatype.weight_port_ids[0]
+    onnx_graph = ONNXGraph(model)
+    allowed_types_list = ["TensorProto.FLOAT", "TensorProto.DOUBLE"]
```

Comment:
TensorProto.DOUBLE also is not supported for quantization. Take a look at https://github.com/openvinotoolkit/nncf/blob/develop/nncf/onnx/graph/nncf_graph_builder.py#L327
So, I recommend removing it from the list for now and putting a comment or creating a ticket.